uahf-technical-spec.md

```diff
@@ -208,6 +219,15 @@ digest based on BIP143. Revisions were made to account for non-Segwit
 deployment.
 
 
+### REQ-6-4 (mandatory use of SCRIPT_VERIFY_STRICTENC)
+
+Once the fork has activated, a transaction shall be deemed invalid if
```

Comment:
@tomz : you are correct it's not in the transaction, I must correct. Unfortunately I can't find one single consolidated place for this (BIP62 is withdrawn).

Best description of what it does here seems to be the code itself, starting from the definition in src/script/interpreter.h:

```
// Passing a non-strict-DER signature or one with undefined hashtype to a
// checksig operation causes script failure. Evaluating a pubkey that is not
// (0x04 + 64 bytes) or (0x02 or 0x03 + 32 bytes) by checksig causes script
// failure.
```

Other implementation will hopefully be able to comply with this by following the C++ code here.

I'll amend the wording to make it clearer.